conf/TransformationZvdd-title-print.xml

```diff
@@ -52,7 +52,7 @@
 						<cgxml:literal name="medium" value="http://rdaregistry.info/termList/RDAproductionMethod/1010"/>
 						<cgxml:literal name="fulltextOnline" value="http://digi.ub.uni-heidelberg.de/diglit/falke1873"/>
 						<cgxml:literal name="title" value="Kunst und Kunstgewerbe auf der Wiener Weltausstellung 1873"/>
-						<cgxml:literal name="issued" value="1873"/>
+						<cgxml:literal name="issued" value="07.04.1873[nach dem 07.04.1873]"/>
```

Comment:
The problem would be to have lots (?) of facets with only one hit and you wouldn't hit this one when facetting on `1873`.